README.md

```diff
@@ -46,15 +52,68 @@ connection_parameters = {
 }
 
 session = Session.builder.configs(connection_parameters).create()
-df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
+df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"]) # Snowpark Python dataframe
```

Comment:
might be better to make this comment consistent with the snowpark pandas one, add on top "Create Snowpark dataframe" let's just call this snowpark dataframe instead have "python" inside, i don't think we cal it snowpark python dataframe anywhere

Reply:
Done